src/core/components/button/rollup.config.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
 	],
 	external: [
 		"react",
+		"react/jsx-runtime",
```

Comment:
This is added as otherwise there is an `Unresolved dependencies` warning when building. 

There is currently also a warning for `jsxs,Fragment,jsx imported from external module 'react/jsx-runtime' but never used` whether this is included or not.